raylib/platforms/rcore_android.c

```diff
@@ -134,6 +135,11 @@ struct android_app *GetAndroidApp(void)
     return platform.app;
 }
 
+extern struct ANativeActivity *GetANativeActivity(void)
```

Comment:
reason for this is in pure C we can make #include "rcore_android.h" for get jni, but in golang would be pain ...